crates/frontend/src/pages/default_config.rs

```diff
@@ -480,6 +492,89 @@ pub fn get_bread_crums(key_prefix: Option<String>) -> Vec<BreadCrums> {
     bread_crums
 }
 
+#[component]
+fn default_config_filter_widget(
+    filters_rws: RwSignal<DefaultConfigFilters>,
+    key_prefix: RwSignal<Option<String>>,
+) -> impl IntoView {
+    let filters = filters_rws.get_untracked();
+    let filters_buffer_rws = create_rw_signal(filters.clone());
+    view! {
+        <DrawerBtn
+            drawer_id="default_config_filter_drawer".into()
+            style="cursor-pointer btn btn-purple-outline m-1 w-[8rem]".to_string()
+        >
+            Filters
+            <i class="ri-filter-3-line"></i>
+        </DrawerBtn>
+        <Drawer
+            id="default_config_filter_drawer".to_string()
+            header="default_config Filters"
```

Comment:
shouldn't the header be something like: (you wouldn't want to use snake case in the displayed text in UI)
```suggestion
            header="Default Config Filters"
```